packages/server/graphql/types/JiraServerIntegration.ts

```diff
@@ -162,6 +179,53 @@ const JiraServerIntegration = new GraphQLObjectType<{teamId: string; userId: str
       resolve: async ({teamId, userId}, _args: unknown, {dataLoader}) => {
         return dataLoader.get('allJiraServerProjects').load({teamId, userId})
       }
+    },
+    providerId: {
+      type: new GraphQLNonNull(GraphQLInt),
+      resolve: async ({teamId, userId}, _args: unknown, {dataLoader}) => {
+        const auth = await dataLoader
+          .get('teamMemberIntegrationAuths')
+          .load({service: 'jiraServer', teamId, userId})
+
+        return auth?.providerId
+      }
+    },
+    searchQueries: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(JiraSearchQuery))),
+      description:
+        'the list of suggested search queries, sorted by most recent. Guaranteed to be < 60 days old',
+      resolve: async ({teamId, userId}, _args: unknown, {dataLoader}) => {
+        const auth = await dataLoader
+          .get('teamMemberIntegrationAuths')
+          .load({service: 'jiraServer', teamId, userId})
+
+        if (!auth) {
+          return []
+        }
+
+        const searchQueries = await getLatestIntegrationSearchQueries({
+          teamId,
+          userId,
+          service: 'jiraServer',
+          providerId: auth.providerId
+        })
+
+        return searchQueries.map((searchQuery) => {
+          const query = searchQuery.query as {
+            queryString: string | null
+            isJQL: boolean
+            projectKeyFilters: string[] | null
+          }
+
+          return {
+            id: searchQuery.id,
```

Comment:
-1 This will not be unique, so we need to prefix it. There is already `SearchQueryId.join` but that's Meeting specific.

Reply:
@Dschoordsch it is unique as the `id` comes from postgresql table

Reply:
@Dschoordsch pushed a commit that uses `toGlobalId` and `fromGlobalId` from 'graphql-relay' package. I think this way we can avoid creating a new function every time. What do you think?

Reply:
The issue with the proposal #6642